### PR TITLE
build docker image

### PR DESCRIPTION
**Pull request by slotix**
Following the issue #20 I've built docker image for se-scraper. 

**Build docker image**

```docker build -t se-scraper . ```

Ready-to-use image is available at https://hub.docker.com/r/slotix/se-scraper 

**Run se-scraper.**

```docker run -it -e HOST=0.0.0.0 -e PORT=3000 -p 3000:3000 slotix/se-scraper```

**Curl dockerized se-scraper service.**

```
curl -XPOST http://0.0.0.0:3000 -H 'Content-Type: application/json' \
-d '{
    "user_agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.110 Safari/537.36",
    "random_user_agent":true,
    "sleep_range":"",
"search_engine":"baidu",
    "debug":true,
    "verbose":true,
    "keywords":[ "cat",  "mouse" ],
    "keyword_file":"",
    "num_pages":1,
    "headless":true,
    "chrome_flags":[ ],
    "output_file":"examples/results/baidu.json",
    "block_assets":false,
    "custom_func":"",
    "proxy":"",
    "proxy_file":"",
    "test_evasion":false,
    "apply_evasion_techniques":true,
    "log_ip_address":false,
    "log_http_headers":false,
    "puppeteer_cluster_config":{
        "timeout":600000,
        "monitor":false,
        "concurrency":1,
        "maxConcurrency":1
    }
}'
```

### CODE REVIEW

**Comment by kederrac** [comment]
can you provide also the Dockerfile please

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+libnss3 \
+lsb-release \
+xdg-utils \
+wget
+
+# Bundle app source
+COPY . .
+WORKDIR $puppeteer_cluster_directory
+RUN npm install \
+    && npm run build
```
Comment (tobiasmuehl) [inline Dockerfile]: Fails for me here

Comment (slotix) [reply Dockerfile]: try ``` git submodule update --init --recursive ```  for fetching puppeteer-cluster  
before ```docker build ``` command

Comment (tobiasmuehl) [reply Dockerfile]: Cool, works now

**Comment by slotix** [comment]
Thank you for adding support of docker in master branch. Closing this pull request...

File: examples/google.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "verbose":true,
+    "keywords":["scrapeulous.com", "scraping search engines", "scraping service scrapeulous", "learn js"],
+    "keyword_file":"",
+    "num_pages":2,
+    "headless":true,
+    "chrome_flags":[ ],
+    "output_file":"examples/results/google.json",
+    "block_assets":false,
+    "custom_func":"",
+    "proxy":"http://proxy:24000",
```
Comment (binyoucai) [inline examples/google.json]: If I have an account and password, after my test, the request failed.

**Pull request closed without merging**